docker/cleanspeak/docker-compose.yml

```diff
@@ -37,8 +37,8 @@ services:
     volumes:
       - es_data:/usr/share/elasticsearch/data
 
-  cleanspeak-management-interface:
-    image: cleanspeak/cleanspeak-management-interface:latest
+  cleanspeak:
```

Comment:
Should we update this `docker-compose` a bit further? You can review the FusionAuth one for reference. 

For example:
- Update Postgres
- Update to OpenSearch

Also, this is documented here https://cleanspeak.com/docs/3.x/tech/installation-guide/docker so we'll want to add this to the list of doc tasks when we ship this.

Reply:
Done. And re: doc tasks - yes. Have a running list on Linear already.